README.md

```diff
@@ -59,6 +59,7 @@ $tokenRequestContext = new AuthorizationCodeContext(
 ```
 Note that your application will need to handle redirecting the user to the Microsoft Identity login page to get the `authorization_code` that's passed into the `AuthorizationCodeContext`.
 [See](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow) for more on the `authorization_code` grant flow.
+To prevent signing in your user before each request, see section on [access token management](docs/Examples.md#access-token-management)
```

Comment:
Does this mean that today, every request needs to be authenticated? If this is the case, why aren't we making this option the default one?

Reply:
No. Tokens are cached & reused for requests made on the same `GraphServiceClient` instance.

Reply:
Updated the section [here](https://github.com/microsoftgraph/msgraph-sdk-php/pull/1475/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R62)